src/api/meetings.js

```diff
@@ -31,15 +31,17 @@ export const fetchMeetings = (token, startDate, endDate) => {
     .then(response => response.body)
 }
 
-export const editMeeting = (token, meeting, room) => agent
-  .put(`${apiBaseUrl}/room/${room.email}/meeting/${meeting.id}`)
-  .set('x-access-token', token)
-  .send({
-    title: meeting.title,
-    start: meeting.start,
-    end: meeting.end,
-  })
-  .then(response => response.body)
+export const editMeeting = (token, meeting, roomEmail) => {
```

Comment:
Just as a note, in a previous PR we had modified this to pass in room instead of roomEmail to be consistent with `createMeeting`. Doesn't matter which we do really, but I think we should keep it consistent.